src/Microsoft.VisualBasic/src/Resources/Strings.resx

```diff
@@ -1047,4 +1047,10 @@
   <data name="WebNotSupportedOnThisSKU" xml:space="preserve">
     <value>Current target framework does not support web operations.</value>
   </data>
+  <data name="NoShellCopyOrMove" xml:space="preserve">
+    <value>UI not available for ShellCopyMove</value>
```

Comment:
>not available for ShellCopyMove [](start = 14, length = 31)

Perhaps `UI not available for copy or move`.

Reply:
Fixed